cmd/completion.go

```diff
@@ -14,7 +14,7 @@ func init() {
 // completionCmd represents the completion command
 var completionCmd = &cobra.Command{
 	Use:   "completion <bash|zsh|fish>",
-	Short: "Generate bash/zsh completion scripts",
+	Short: "Generate completion scripts for bash and zsh",
```

Comment:
```suggestion
	Short: "Generate completion scripts for bash, fish and zsh",
```

Reply:
As a `fish` user, I support this change! :)